ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -75,11 +77,78 @@ + (BOOL)validBrokerRedirectUri:(NSString*)url
     return NO;
 }
 
-+ (void)internalHandleBrokerResponse:(NSURL *)response
++ (BOOL)internalHandleBrokerResponse:(NSURL *)response
 {
 #if TARGET_OS_IPHONE
-    ADAuthenticationCallback completionBlock = [ADBrokerHelper copyAndClearCompletionBlock];
-    HANDLE_ARGUMENT(response, nil);
+    __block ADAuthenticationCallback completionBlock = [ADBrokerHelper copyAndClearCompletionBlock];
+        
+    ADAuthenticationResult* result = [self processBrokerResponse:response];
+    
+    [[NSUserDefaults standardUserDefaults] removeObjectForKey:kAdalResumeDictionaryKey];
```

Comment:
Nope. openURL will get hit first if we're hit by a URL. In other cases, if another broker request happens it'll overwrite the dictionary, and if we end up getting hit with a broker response afterwards somehow the data could be useful for dropping it in the proper cache location. (If we choose to do such a thing) So in either case leaving around the data isn't harmful and possibly useful.